lineman/app/components/previous_proposals_page/previous_proposals_page.coffee

```diff
@@ -0,0 +1,11 @@
+angular.module('loomioApp').controller 'PreviousProposalsPageController', ($scope, $rootScope, $routeParams, Records) ->
+  $rootScope.$broadcast('currentComponent', { page: 'previousProposalsPage'})
+
+  Records.groups.findOrFetchById($routeParams.key).then (group) =>
```

Comment:
We'll want a permissions check on here to make sure you can view stuff for this group; similar to `memberships_page_controller.coffee`, I reckon.